app/controllers/quick_reference_controller.rb

```diff
@@ -8,7 +8,8 @@ class QuickReferenceController < ApplicationController
     'pipelines/wait_step',
     'pipelines/block_step',
     'pipelines/input_step',
-    'pipelines/trigger_step'
+    'pipelines/trigger_step',
+    'pipelines/managing-priorities'
```

Comment:
This isn't a step, so it probably doesn't belong here! Linking it from the command step is sufficient, and doesn't require an icon 👍🏼